### ISSUE TITLE
Error: Error trying to ping. Error: Composer runtime (0.16.3) is not compatible with client (0.16.2)

### ISSUE THREAD

**Issue by sahilsehgalinc** [hyperledger/composer#3387]
After setting up Multi Organization tutorial as mentioned below link, I see following error message on Hyperledger Composer:

https://hyperledger.github.io/composer/tutorials/deploy-to-fabric-multi-org

Error: Error trying to ping. Error: Composer runtime (0.16.3) is not compatible with client (0.16.2)

I also downgraded to the version from 0.16.3 to 0.16.2:

root@ldap:~/fabric-tools/fabric-samples/first-network# composer -v
v0.16.2
root@ldap:~/fabric-tools/fabric-samples/first-network# composer-rest-server -v
v0.16.2

Not able to understand where can be the issue. Can anybody help on this?

**Comment by davidkel** [comment]
The client version must be either the same or higher version than the runtime. Your client application must be using 0.16.2 of the composer sdk or the CLI you are using is at 0.16.2. However when you deployed your business network you must have used 0.16.3 to do so because this is the level of composer runtime that got deployed to the fabric. The solution is to upgrade your CLI or your application to use 0.16.3 or higher if a newer version is available.

**Comment by nitishbhushan2013** [comment]
I also faced the same problem and found an issue .. 

Problem
----------
while executing below command got an error
composer-rest-server -c admin@airlinev8 -p 3000 

Nitishs-MacBook-Pro:fabric-dev-servers nitishbhushan$ composer-rest-server -c admin@airlinev8 -p 3000 
Discovering types from business network definition ...
Connection fails: Error: Error trying to ping. Error: Composer runtime (0.19.9) is not compatible with client (0.19.8)
It will be retried for the next request.
Exception: Error: Error trying to ping. Error: Composer runtime (0.19.9) is not compatible with client (0.19.8)
Error: Error trying to ping. Error: Composer runtime (0.19.9) is not compatible with client (0.19.8)

solution
---------
step 1: check version of composer-cli and composer-rest-server
composer-cli - 0.19.9
composer-rest-server - 0.19.8

step2: install latest version of composer-rest-server 
npm install composer-rest-server -g

After this step, I ran the command again and it worked  :-)
